src/art/local/tokenize.py

```diff
@@ -57,13 +57,27 @@ def tokenize_trajectory_groups(
             # Skip trajectories with no advantage
             if advantage == 0:
                 continue
-            if result := tokenize_trajectory(
-                tokenizer,
-                trajectory,
-                advantage,
-                allow_training_without_logprobs,
-            ):
-                results.append(result)
+            trajectory_results: list[TokenizedResult] = []
+            for history in [
```

Comment:
Probably doesn't matter but we could make `Trajectory` inherit from `History` to avoid having to create a new one here.